host/xtest/xtest_4000.c

```diff
@@ -4235,3 +4247,129 @@ static void xtest_tee_test_4008(ADBG_Case_t *c)
 	Do_ADBG_EndSubCase(c, "Derive DH key success");
 	TEEC_CloseSession(&session);
 }
+
+static void xtest_tee_test_4009(ADBG_Case_t *c)
+{
+	TEEC_Session session = { 0 };
+	uint32_t ret_orig;
+	TEE_OperationHandle op;
+	TEE_ObjectHandle key_handle;
+	TEE_ObjectHandle sv_handle;
+	TEE_Attribute params[4];
+	size_t param_count = 0;
+	uint8_t out[2048];
+	size_t out_size;
+	uint32_t size_bytes;
+	int i;
+	struct derive_key_ecdh_t *pt = &(derive_key_ecdh[0]);
+
+	if (!ADBG_EXPECT_TEEC_SUCCESS(c,
+		xtest_teec_open_session(&session, &crypt_user_ta_uuid, NULL,
+					&ret_orig)))
+		return;
+
+	for (i = 0; i < ARRAY_SIZE(derive_key_ecdh); i++) {
+		pt = &(derive_key_ecdh[i]);
```

Comment:
A complicated way of writing `pt = derive_key_ecdh + i`. I know it's a matter of taste, but at least the parenthesis could be removed.